### PR TITLE
Can you check it, please.

### PR DESCRIPTION
**Pull request by ayumi-watanabe**
Can you check it, please

### CODE REVIEW

File: app/views/static_pages/home.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<% provide(:title, 'Home') %>
+　<div class="jumbotron">
+<nav class="navbar navbar-dark bg-primary">
+  <!-- Navbar content -->
+</nav>
+  <h1 class="text-center display-4">Welcome!</h1>
+  <p class="lead"></p>
+  <hr class="my-4">
+  <p>Here is a space where you can shere the thing what you want.</p>
+</div>
```
Comment (aplazand20) [inline app/views/static_pages/home.html.erb]: Change from Spaces: 2 to Spaces: 4

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,15 @@
 Rails.application.routes.draw do
+  get 'home/top'
+  get 'static_pages/new'
+  get 'static_pages/index'
+  get 'static_pages/home'
+  get 'static_pages/about'
+  get 'static_pages/contact'
```
Comment (aplazand20) [inline config/routes.rb]: You can delete these lines of code since we don't need it.

File: data/ib_buffer_pool [hunk 3]
```diff
@@ -1,210 +1,109 @@
-3,3
-0,51
+3,2
```
Comment (aplazand20) [inline data/ib_buffer_pool]: **Do this one last**

image
Add these lines of code to your .gitignore

After you are done editing everything, follow these commands:
`git rm -r --cached data`

`git add .`

`git commit -m "Fixing data and fixing spaces"`

`git push -u origin static_pages`

**Review by aplazand20 — COMMENTED** [review]
Please do these changes

**Comment by ayumi-watanabe** [comment]
Can you check again, please.

**Pull request merged**